lib/Utils/Architectures.pm

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package Utils::Architectures;
+use strict;
+use warnings;
+
+use base qw(Exporter);
+use Exporter;
+use testapi qw(:DEFAULT);
```

Comment:
only check_var is used here, there's no need to import the full testapi :) at least for now.

Reply:
yes, changed, thanks